pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -654,6 +654,16 @@ func DefaultOpenAPIConfig(config configapi.MasterConfig) *openapicommon.Config {
 			},
 		}
 	}
+	if config.OAuthConfig != nil {
+		securityDefinitions["Oauth2AccessToken"] = &spec.SecurityScheme{
+			SecuritySchemeProps: spec.SecuritySchemeProps{
+				Type:             "oauth2",
+				Flow:             "accessCode",
+				AuthorizationURL: config.OAuthConfig.MasterURL + "/oauth/authorize",
```

Comment:
Something like this would work (hopefully there is not an import cycle):

```go
...
AuthorizationURL: OpenShiftOAuthAuthorizeURL("https://localhost:8443"),
TokenURL: OpenShiftOAuthTokenURL("https://localhost:8443"),
...
```